tests/test_gamma.py

```diff
@@ -363,3 +363,54 @@ def test_meta_data_exists(series_ifgs, parallel_ifgs):
         assert (s.meta_data[ifc.DATA_TYPE] == ifc.MULTILOOKED) or \
                (s.meta_data[ifc.DATA_TYPE] == ifc.MULTILOOKED_COH)
     assert i + 1 == 34
+
+
+class TestGammaBaselineRead:
+    """Tests the reading of initial and precise baselines"""
+
+    def setup_method(self):
+        init_path = join(GAMMA_TEST_DIR, '20160114-20160126_base_init.par')
+        self.init = gamma.parse_baseline_header(init_path)
+        prec_path = join(GAMMA_TEST_DIR, '20160114-20160126_base.par')
+        self.prec = gamma.parse_baseline_header(prec_path)
+
+    def test_prec_baseline_read(self):
+        """Test that the Precise baseline values are being read"""
+        exp_i = {'BASELINE_T': -0.0000026, 'BASELINE_C': -103.7427072,
+                 'BASELINE_N': 2.8130731, 'BASELINE_RATE_T': 0.0,
+                 'BASELINE_RATE_C': -0.0173538,
+                 'BASELINE_RATE_N': -0.0055098}
+
+        exp_p = {'BASELINE_T': 0.0, 'BASELINE_C': -103.8364725,
+                 'BASELINE_N': 2.8055662, 'BASELINE_RATE_T': 0.0,
+                 'BASELINE_RATE_C': -0.0182215,
+                 'BASELINE_RATE_N': -0.0065402}
+
+        # Precise values are read
+        assert self.prec != exp_i
+        assert self.prec == exp_p
```

Comment:
Should this test pass? The comment in `parse_baseline_header` suggests that precise values are ignored unless _all_ values are non-zero.